dry-view.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.1.0'
 
+  spec.add_runtime_dependency "inflecto"
```

Comment:
In all other gems we depend on dry-core and its `Dry::Core::Inflector` module, any specific reason why dry-view should depend directly on inflecto?